packages/engine/Specs/Renderer/ContextSpec.js

```diff
@@ -325,6 +327,36 @@ describe(
       expect(c.drawingBufferHeight).toBe(768);
       c.destroyForSpecs();
     });
+
+    it("requestWebgl1 works", function () {
+      const c1 = createContext({
+        requestWebgl1: true,
+      });
+      expect(c1._webgl2).toBe(false);
+
+      if (!webglStub) {
+        const c2 = createContext({
+          requestWebgl1: false,
+        });
+        expect(c2._webgl2).toBe(true);
+      }
+    });
+
+    it("prints deprecation warning when using requestWebgl2 instead of requestWebgl1", function () {
```

Comment:
Great that this was tested explicitly, but typically there are no explicit tests for deprecation warnings as they will need to be removed.

Reply:
Do you recommend removing this spec or should I create an issue to `remove in 1.102` and address it the WebGL2 by default PR? I also wanted to add a spec to ensure the now deprecated `requestWebgl2` was selecting the context correctly.

Reply:
I think remove it. And the WebGL2 by default PR should be testing this new option instead.

Reply:
Updated.